14_contains/solution/contains-solution.spec.js

```diff
@@ -0,0 +1,45 @@
+const contains = require('./contains-solution');
+
+describe('contains', () => {
+  const nestedObject = {
```

Comment:
```suggestion
  const object = {
```
`nestedObject` implies *that* object is nested inside something, but it's not.